RM_SUI/RM_SUI/ShowcaseShowroomView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ShowcaseShowroomView.swift
+//  RM_SUI
+//
+//  Created by Дмитрий Супрун on 16.01.23.
+//
+
+import SwiftUI
+
+/// Showcase
+struct ShowcaseShowroomView: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let emptyString = ""
+        static let logoImageName = "GeelyLogo"
+        static let showcaseImageWidthHeight: CGFloat = 300
+        static let showcaseImageCornerRadius: CGFloat = 50
+        static let showcaseImageShadowRadius: CGFloat = 15
+        static let showcaseImageShadowX: CGFloat = 5
+        static let showcaseImageShadowY: CGFloat = 5
+        static let fuelTypeName = "Fuel: "
+        static let transmissionTypeName = "Transmission: "
+        static let enginePower = "Engine Power: "
+        static let shareButtonImageName = "square.and.arrow.up"
+        static let shareButtonImageWidthHeight: CGFloat = 30
+        static let imageZoomRange: ClosedRange<CGFloat> = 150...300
+        static let pickerName = "Car"
+    }
+    
+    // MARK: - Body
+    var body: some View {
+        NavigationView {
+            VStack {
+                showcaseLogoImage
+                ZStack {
+                    backgroundCarImageRectangle
+                    carImage
+                }
+                carSelectionPicker
+                Form {
+                    fuelInfoText
+                    Picker("Transmission", selection: $transmissionChange) {
+                        Text("automatic")
+                        Text("manual")
+                    }.pickerStyle(.navigationLink)
+                    powerInfoText
+                    sizeSlier
+                }
+                shareButton
+            }
+        }
+    }
+    
+    // MARK: Private Properties
+    
+    private let showroom = Showroom()
+    @State private var selectionCarPicker = 0
+    @State private var isShareSheetPresented = false
+    @State private var imageWidthHeight = Constants.showcaseImageWidthHeight
+    @State private var transmissionChange = "automatic"
+    
+    private var showcaseLogoImage: some View {
+        Image(Constants.logoImageName)
+            .resizable()
+            .scaledToFit()
+            .padding()
+    }
+    private var backgroundCarImageRectangle: some View {
```

Comment:
```suggestion

    private var backgroundCarImageRectangle: some View {
```
не слепливай свойства